src/app/lib/constants/styles.js

```diff
@@ -38,6 +38,16 @@ export const GEL_SPACING_TRPL = `1.5rem`;
 export const GEL_SPACING_HLF = `0.25rem`;
 export const GEL_SPACING_QUAD = `2rem`;
 
+/*
+    GEL Grid
+    Margins and Gutters are defined here
+    https://www.bbc.co.uk/gel/guidelines/grid#grid-sizes
+*/
+export const GEL_MARGIN_BELOW_400PX = `${GEL_SPACING}`;
```

Comment:
Do these need to be variables, can we just use `GEL_SPACING` or `GEL_SPACING_DBL` directly?

Reply:
@jtart @ChrisBAshton 
The GEL SPACING variables were initially used directly. These `GEL_MARGIN_*` `GEL_GUTTER_*` values however *are* constants that GEL Grid has given us. (https://www.bbc.co.uk/gel/guidelines/grid#grid-sizes)
It makes the code in the `layoutGrid.js` file easier to read and removes the need for suggested comments like `// UX gutter` `// UX margin exclusive of grid` as explanations. It is self-documenting.

Reply:
I like self-documenting 👍